FreeRTOS/coreJSON
# Restructure function contracts for readability

Description
-----------
Restructure function contracts for readability.

Test Steps
-----------
All CBMC harnesses should continue to pass. No fundamental changes are being applied.

Checklist:
----------



Related Issue
-----------
None.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
